README.md

```diff
@@ -26,9 +26,9 @@ If your applications aren't sending traces, yet, configure them with [Zipkin ins
 Check out the [`zipkin-server`](/zipkin-server) documentation for configuration details, or [`docker-zipkin`](https://github.com/openzipkin/docker-zipkin) for how to use docker-compose.
 
 ## Core Library
-The [core library](zipkin/src/main/java/zipkin2) is used by both Zipkin instrumentation and the Zipkin server. Its minimum Java language level is 6, in efforts to support those writing agent instrumentation.
+The [core library](zipkin2/src/main/java/zipkin2) is used by both Zipkin instrumentation and the Zipkin server. Its minimum Java language level is 6, in efforts to support those writing agent instrumentation.
 
-This includes built-in codec for Zipkin's v1 and v2 json formats. A direct dependency on gson (json library) is avoided by minifying and repackaging classes used. The result is a 155k jar which won't conflict with any library you use.
```

Comment:
This is incorrect change. the new library still supports encoding to v1 format, and it is used today https://github.com/openzipkin/zipkin/blob/master/zipkin2/src/main/java/zipkin2/codec/SpanBytesEncoder.java#L27